features/full/detect_anr_cxx.feature

```diff
@@ -0,0 +1,27 @@
+Feature: ANRs triggered in CXX code are captured
+
+@skip_android_8_1
+Scenario: ANR triggered in CXX code is captured
+    When I run "CXXAnrScenario"
+    And I wait for 2 seconds
+    And I tap the screen 3 times
+    And I wait for 4 seconds
+    And I clear any error dialogue
+    Then I wait to receive a request
+    And the request is valid for the error reporting API version "4.0" for the "Android Bugsnag Notifier" notifier
+    And the exception "errorClass" equals "ANR"
+    And the exception "message" starts with " Input dispatching timed out"
+    And the exception "type" equals "c"
```

Comment:
As per comment above - remove this and we'll add the native assertions once the frame-level functionality is merged.

Reply:
Removed assertion for now